### PR TITLE
R example

### PR DESCRIPTION
**Pull request by shilohd**
Related to issue #44. I've added new R examples where they were missing in the configuration and core programming pages.

### CODE REVIEW

File: book/configuration.md [hunk 1]
```diff
@@ -22,8 +24,32 @@ model.fit(variables_train, outcome_train)
 prediction = model.predict(variables_test, constant_a=7, max_v=1000)
 
 prediction.to_csv("outputs/predictions.csv")
+
+```
+
+```{code-tab} r R
+data <- read.csv("C:/a/very/specific/path/to/input_data.csv")
```
Comment (alexander-newton) [inline book/configuration.md:35]: add a note to top of each saying that it's not good practice

File: book/configuration.md [hunk 2]
```diff
@@ -59,6 +86,33 @@ prediction = model.predict(variables_test, constant_a=prediction_parameters["con
 prediction.to_csv(output_path)
 ```
 
+```{code-tab} r R
+# Configuration
```
Comment (alexander-newton) [inline book/configuration.md:98]: again, add a note saying it's not good practice

File: book/core_programming.md [hunk 3]
```diff
@@ -1102,6 +1107,25 @@ def extended_methodology(data):
...
+  ...
+  return(result)
+}
+
+
+# if we want to extend a function without modifying it, we can always do the following
+extended_methodology <- function(data){
+  core_results = core_method(data)
+  return(
+    extended_functionality(core_results)
```
Comment (alexander-newton) [inline book/core_programming.md:1135]: can you also define this function in the example?

**Review by alexander-newton — CHANGES_REQUESTED** [review]
some changes then happy

File: book/configuration.md (hunk 1 above)
Comment (shilohd) [reply book/configuration.md:35]: Fixed.

File: book/configuration.md (hunk 2 above)
Comment (shilohd) [reply book/configuration.md:98]: Fixed.

File: book/core_programming.md (hunk 3 above)
Comment (shilohd) [reply book/core_programming.md:1135]: Fixed.

**Review by alexander-newton — APPROVED** [review]

**Pull request merged**